test/cypress/integration/widgets/filters_spec.ts

```diff
@@ -370,7 +370,7 @@ describe('Filters widget', () => {
         });
     });
 
-    describe('should allow to define a filter rule', () => {
+    describe.skip('should allow to define a filter rule', () => {
```

Comment:
Minor:

I would appreciate adding a TODO comment

```suggestion
    // TODO(RD-2664): fix the double-fetch in DynamicDropdown and enable the tests
    describe.skip('should allow to define a filter rule', () => {
```